pom.xml

```diff
@@ -118,6 +118,14 @@
         </dependency>
 
         <dependency>
+            <!-- For logging on Windows -->
+            <groupId>org.fusesource.jansi</groupId>
+            <artifactId>jansi</artifactId>
+            <version>1.13</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I got excited when i saw this thinking the JANSI errors in AppVeyor would disappear. Sadly, it seems like they're still there :(

```
15:57:16,246 |-INFO in ch.qos.logback.core.ConsoleAppender[STDOUT] - Enabling JANSI WindowsAnsiOutputStream for the console.
15:57:16,340 |-WARN in ch.qos.logback.core.ConsoleAppender[STDOUT] - Failed to create WindowsAnsiOutputStream. Falling back on the default stream. ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type org.fusesource.jansi.WindowsAnsiOutputStream
	at ch.qos.logback.core.util.DynamicClassLoadingException: Failed to instantiate type org.fusesource.jansi.WindowsAnsiOutputStream
	at 	at ch.qos.logback.core.util.OptionHelper.instantiateByClassNameAndParameter(OptionHelper.java:73)
	at 	at ch.qos.logback.core.util.OptionHelper.instantiateByClassNameAndParameter(OptionHelper.java:42)
	at 	at ch.qos.logback.core.ConsoleAppender.getTargetStreamForWindows(ConsoleAppender.java:94)```

(snippet from https://ci.appveyor.com/project/AlejandroRivera/embedded-rabbitmq/build/1.0.79-master-ingmjqwm)